src/Xamarin.Android.Build.Tasks/Microsoft.Android.Sdk/targets/Microsoft.Android.Sdk.AssemblyResolution.targets

```
@@ -168,8 +168,13 @@ _ResolveAssemblies MSBuild target.
     <ItemGroup>
       <_ResolvedNativeLibraries Include="@(ResolvedFileToPublish)" Condition=" '%(ResolvedFileToPublish.Extension)' == '.so' " />
     </ItemGroup>
+    <ItemGroup>
+      <_MonoComponent Condition=" '$(AndroidEnableProfiler)' != 'false' " Include="diagnostics_tracing" />
+      <_MonoComponent Condition=" '$(AndroidEnableHotReload)' != 'false' " Include="hot_reload" />
```

Maybe we don't need new MSBuild property for hot reload yet, maybe we can do:
```suggestion
      <_MonoComponent Condition=" '$(AndroidIncludeDebugSymbols)' == 'true' and '$(AndroidUseInterpreter)' == 'true' " Include="hot_reload" />
```
See: https://github.com/xamarin/xamarin-android/pull/6027